test/completeness/test_completeness_pipe.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+
+
+# Test input check on real data
+def test_1_load_data(data_dic):
```

Comment:
if using conftest, the arguments need to be the same as the function name, and the value is the value returned